### PR TITLE
Update estimated completed date logic for claims status

### PR DESCRIPTION
**Pull request by jbalboni**
Closes [121](https://github.com/department-of-veterans-affairs/sunsets-team/issues/121)

This updates the estimated date logic according to the above issue.

Two things still missing:
- Link for explanation (separate issue)
- Text about estimated date changing (waiting for more information)

screen shot 2016-10-25 at 12 07 21 pm
screen shot 2016-10-25 at 12 06 38 pm
screen shot 2016-10-25 at 1 08 21 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@austin-martinez or @cagedcrown, can you take a look?

File: src/js/disability-benefits/components/ClaimEstimate.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import moment from 'moment';
+
+export default function ClaimEstimate({ maxDate }) {
+  const estimatedDate = moment(maxDate);
+  const today = moment().startOf('day');
+
+  if (!maxDate || !estimatedDate.isValid() || estimatedDate.isAfter(moment(today).add(2, 'years'))) {
```
Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimEstimate.jsx]: Do you need to check `!maxDate` here? You've already wrapped `maxDate` in moment, so it seems like any validity checks should use the wrapped variable, or that `maxDate` shouldn't be wrapped if it is already invalid

Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimEstimate.jsx]: Also, `today` is already a moment object, do you need to wrap it again to add 2 years?

**Review by austin-martinez — APPROVED** [review]
Overall LGTM :framed_picture: 2 minor questions

**Pull request merged**